src/probeinterface/utils.py

```diff
@@ -47,7 +47,7 @@ def import_safely(module: str) -> ModuleType:
     return module_obj
 
 
-def combine_probes(probes: Probe, connect_shape: bool = True) -> Probe:
+def combine_probes(probes: list[Probe], connect_shape: bool = True) -> Probe:
```

Comment:
@zm711 in general, I believe that `list` is supported from Python3.9, otherwise we need `from __future__ import annotations`.

Since 3.8 will be alive until end of next year, let's be compliant with that.

Reply:
Done.